doc/man3/SSL_key_update.pod

```diff
@@ -0,0 +1,119 @@
+=pod
+
+=head1 NAME
+
+SSL_key_update,
+SSL_get_key_update_type,
+SSL_renegotiate,
+SSL_renegotiate_abbreviated,
+SSL_renegotiate_pending
+- initiate and obtain information about updating connection keys
+
+=head1 SYNOPSIS
+
+ #include <openssl/ssl.h>
+
+ /* TLSv1.3 KeyUpdate message types */
+ typedef enum {
+     /* -1 used so that this is an invalid value for the on-the-wire protocol */
```

Comment:
Hmmm. If we're going to say that, then probably we should not also use the enum for the input parameter to SSL_key_update(). They are the same thing so they should be the same type - if one is an enum then so is the other. If one is an int then they are both ints. That doesn't stop us from treating it as an enum internally of course.